packages/api/tests/insightsAPI.test.js

```diff
@@ -270,6 +269,9 @@ xdescribe('insights test', () => {
         done();
       });
 
+      // FIXME: these tests are incomplete and skipped, seems a mocks.cropSaleFactory
```

Comment:
I think the method is called `crop_variety_saleFactory`, but there are other things going wrong here beyond the mock name 🙂